### PR TITLE
change jdbc clickhoue verstion into 0.3.2-patch9

### PR DESCRIPTION
**Pull request by grishick**
Created a non-forked version of [this PR](https://github.com/airbytehq/airbyte/pull/13639) to run CI

### CODE REVIEW

**Review by ryankfu — APPROVED** [review]
LGTM, I see the old PR the community member did was about a month and half old so it seems that Clickhouse has updated to `0.3.2-patch10`. I would consider upgrading to the newest version since it appears we don't have any specific reason not to unless I'm mistaken:  https://github.com/ClickHouse/clickhouse-jdbc/releases/tag/v0.3.2-patch10

File: docs/integrations/destinations/clickhouse.md [hunk 1]
```diff
@@ -79,8 +79,9 @@ Therefore, Airbyte ClickHouse destination will create tables and schemas using t
 
 | Version | Date       | Pull Request | Subject                                      |
 |:--------|:-----------| :--- |:---------------------------------------------|
-| 0.1.8 | 2022-07-05 | [\#13516](https://github.com/airbytehq/airbyte/pull/13516) | Added JDBC default parameter socket timeout |
-| 0.1.7 | 2022-06-16 | [\#13852](https://github.com/airbytehq/airbyte/pull/13852) | Updated stacktrace format for any trace message errors |
+| 0.1.10  | 2022-07-05 | [\#13639](https://github.com/airbytehq/airbyte/pull/13639) | Change JDBC ClickHouse version into 0.3.2-patch9 |
```
Comment (ryankfu) [inline docs/integrations/destinations/clickhouse.md:82]: Is the reason we jump to `0.1.10` to have parity between non `strict-encrypt` and `strict-encrypt` versions?

Comment (grishick) [reply docs/integrations/destinations/clickhouse.md:82]: yes

**Comment by grishick** [comment]
/publish connector=connectors/destination-clickhouse,connectors/destination-clickhouse-strict-encrypt

**Pull request merged**